ansible/vars/topo_smartswitch.yml

```diff
@@ -0,0 +1,669 @@
+topology:
```

Comment:
@nissampa could you please rename the file to be like topo_smartswitch-t1.yml? So it will be easier to name the t0 smartswitch topo in future. And we also need the symbolic links for the eos VM tor and leaf template, like ansible/roles/eos/templates/t1-28-lag-tor.j2 and ansible/roles/eos/templates/t1-28-lag-leaf.j2.

Reply:
Changed the topo name to topo_smartswitch-t1.
Added symbolic links as well. 

Please let me know if there is anywhere I need to add topo in any file.